src/gen_module.sh

```diff
@@ -33,6 +34,20 @@ if [ ! -e ${blender_bin} ]; then
     exit 1
 fi
 
+# check if PYTHON_BIN binary is availble
+if ! command -v ${PYTHON_BIN} > /dev/null; then
+    echo "Error: Cannot find ${PYTHON_BIN} binary."
+    exit 1
+fi
+python=$(command -v ${PYTHON_BIN})
```

Comment:
I think environment variable should not be same name as command name (I think `python` is used for symbolic link to `python2` in many system).
Please rename to `python_bin` or else.

Reply:
Creating a variable `python` in bash, does not "overwrite" a command, it simply creates a variable which can then later be used by `${python}`. It has absolutely no impact on the command `python` which bash evaluates using `$PATH` and executes in case it has been found.

For your convenience, I renamed it anyway.

Reply:
> Creating a variable python in bash, does not "overwrite" a command, it simply creates a variable which can then later be read by using ${python}.

Yes, you are right. I knew this specification.
But I think it is better to recognize between command and environment variable.
Thanks for tackling to fix it.